### DISCUSSION TITLE
How to verify my Syphon Session on other MAtrix Clients

### DISCUSSION THREAD

**Discussion by Mannshoch** [syphon-org/syphon#657] [Q&A] [answered]
Syphon 0.2.11 from Fdroid

How do I verify my Session to my other Matrix clients? Some Channels are currently not readable.

**Comment by notramo** [comment, marked as answer]
https://github.com/syphon-org/syphon/blob/9c570e04929ab0019d5a5b8b1a059090cfe19878/docs/faq-en.md#how-do-i

**Reply by Mannshoch** [reply]
The following point is missing:
5. Click on the untrusted session
6. Click "Manually Verify By Text"

**Comment by 0323pin** [comment]
Hmm ... this doesn't seem to work anymore. Element no longer allows manual verification 😞 

Is there any other way to verify my Syphon session? I can't find "accept verification request" anywhere in the mobile app.
Thanks

**Reply by dnisbetjones** [reply]
You'll need to manually verify for the time being using Element web/desktop. If you tap your name in a chat window, you'll see your sessions appear in the sidebar. From there, you can find the Syphon session, click it and 'Manually verify by text' will appear.

**Reply by 0323pin** [reply]
It works, thanks!
